app/views/RoomView/index.js

```diff
@@ -798,6 +798,13 @@ class RoomView extends React.Component {
 				</View>
 			);
 		}
+		if (room.archived) {
```

Comment:
Instead of create a new case, you can add `room.archived` [here](https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/cd7ab896bbcad9b599e65789e670f4293013c4d8/app/views/RoomView/index.js#L794) and show the same message of web client.
Thanks.